.gitignore

```diff
@@ -72,12 +72,16 @@ venv
 Thumbs.db
 
 # Ignore built javascript files
-/contentcuration/*/static/js/bundles
-/contentcuration/*/static/studio
+/contentcuration/contentcuration/static/js/bundles
+/contentcuration/contentcuration/static/studio
 
-# Ignore Django collected static files
+# The following line doesn't do anything since currently `collectstatic` writes to /contentcuration/contentcuration/static
 /contentcuration/static/*
-contentcuration/contentcuration/static
+
+# Ignore Django collectstatic output directories
```

Comment:
I can approve once we remove these lines from .gitignore. These files shouldn't be in here, and if they are, it can become a problem. (If, say, `rest_framework` gets updated and has newer static files, Django could end up picking up the old copied version in here instead, leading to strange, hard-to-debug errors caused by the code and static files being different versions.)